cdap-security-spi/src/main/java/co/cask/cdap/security/spi/authorization/AbstractAuthorizer.java

```diff
@@ -64,6 +65,22 @@ public void enforce(EntityId entity, Principal principal, Action action) throws
   }
 
   @Override
+  public Set<? extends EntityId> isVisible(Set<? extends EntityId> entityIds, Principal principal) throws Exception {
+    Set<EntityId> visibleEntities = new HashSet<>();
+    for (EntityId entityId : entityIds) {
+      try {
+        enforce(entityId, principal, EnumSet.allOf(Action.class));
```

Comment:
I have `InmemoryAuthorizer` implements this method, for this abstract class, I think we should keep it consistent with the `createFilter` method